package/setup.py

```diff
@@ -341,9 +341,11 @@ def extensions(config):
                         extra_compile_args=extra_compile_args)
     cutil = MDAExtension('MDAnalysis.lib._cutil',
                          sources=['MDAnalysis/lib/_cutil' + source_suffix],
+                         language='c++',
                          include_dirs=include_dirs,
                          define_macros=define_macros,
-                         extra_compile_args=extra_compile_args)
+                         extra_compile_args=[a for a in extra_compile_args
+                                             if not a.startswith('-std')])
```

Comment:
so c++ didn't play well with the `-std=c99` flag.  What does this do? And does removing it break anything?

Reply:
This flag sets the C standard we are using. Yeah, c++ doesn't have this standard. Having an explicit standard for the C-code is nice. I did a PR a while ago where I did small clean up in the C code and I would like to keep the explicit standard requirement.

Reply:
None of the functions I used make use of modern c++ features. If we want to have the compatibility with very old compilers gcc 4.8 and smaller and therefore almost all linux installations out there we can go for the 2003 standard version. Generally I prefer the c++11/14  standard aka modern C++ but we don't make use of any features of those just yet.